files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml

```diff
@@ -1008,7 +1008,7 @@ spades: {cores: 20, mem: 400}
 spyboat: {cores: 8, mem: 4}
 sshmm: {mem: 16}
 structurefold: {mem: 12}
-rnaspades: {cores: 10, mem: 90}
+rnaspades: {cores: 10, mem: 300}
```

Comment:
Wah ... that is a lot :)

How have you figured the usage out? Is that a very large dataset?

Reply:
We got an error report from a user; according to it, it was required approx. 286.433GB of free RAM to assemble the dataset.

Reply:
Wow! another reason we should work on the resource-estimation ML project .... @mtekman @anuprulez

Reply:
Small question -- In the case that we perform metrics, and discover that 99% of users only need 40GB for rnaspades, how would we then handle 1% of users who need 300?

Reply:
This here is a static assignment, but Galaxy supports dynamic assignments. So when a job is submitted Galaxy can ask a python function to return a memory estimation and that estimation can take the input data size into account.

Reply:
Oh I see -- thanks!